cmd/cmd_utils.go

```diff
@@ -186,10 +221,9 @@ func preCustomCommand(
 			os.Exit(1)
 		} else {
 			// truly invalid, nothing to do
-			u.LogError(errors.New(
-				"invalid command: no args, no steps, no sub-commands",
-			))
-			os.Exit(1)
+			u.PrintErrorMarkdownAndExit("Invalid command", errors.New(
+				"no args, no steps, no sub-commands",
+			), "")
```

Comment:
I don't know what this error means.

Reply:
Interesting. I will dig into what it is 😅

Reply:
Made it more useful. It was basically an error when custom command is not valid

Reply:
Done